app/components/zitting-link.js

```diff
@@ -17,7 +17,10 @@ export default class ZittingLinkComponent extends Component {
   @restartableTask
   * getMeeting(){
     const result = yield this.store.query("zitting", {
-      'filter[agendapunten][behandeling][document-container][:id:]':this.args.documentContainer.id
+      'filter[agendapunten][behandeling][document-container][:id:]': this.args.documentContainer.id,
+      // Including the relationships ensures the cache returns the proper response.
+      // TODO: This is a workaround for a mu-cache issue. Remove the include once that's resolved.
+      'include': 'agendapunten.behandeling.document-container'
```

Comment:
I think/hope that perhaps just including agendapunten may be enough (and will result in a smaller payload) :pray:

Reply:
Yup, that seems to work as well! I misunderstood and assumed the whole tree was needed.